client/src/inside/defs-grid/index.js

```diff
@@ -0,0 +1,13 @@
+import patientProfile from '../../inside/defs-grid/patient-profile'
+import currentVisit1 from '../../inside/defs-grid/current-visit-1'
+import currentVisit2 from '../../inside/defs-grid/current-visit-2'
+import patientChart from '../../inside/defs-grid/patient-chart'
+import externalResources from '../../inside/defs-grid/external-resources'
+import testPage from '../../inside/defs-grid/test-page'
+
+export const PP = patientProfile
+export const CV1 = currentVisit1
+export const CV2 = currentVisit2
+export const PC = patientChart
+export const ER = externalResources
+export const TP = testPage
```

Comment:
ideally one CR at end of file please.

Could this module export the concatenated page defs?